scalatest/src/main/scala/org/scalatest/run.scala

```diff
@@ -130,7 +130,7 @@ object run {
    * <code>fullStacksPassed</code>, and <code>statsPassed</code>).
    * </p>
    */
-  def apply(suite: Suite, testName: String = null, configMap: ConfigMap = ConfigMap.empty): Unit = {
+  def apply(suite: Suite, testName: String | Null = null, configMap: ConfigMap = ConfigMap.empty): Unit = {
```

Comment:
**Nullable parameter**
Parameter that can be null and should therefore have a `| Null` type.